packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -257,3 +257,10 @@
     text-align: center;
     width: var(--x-axis-label-width, 20px);
 }
+
```

Comment:
Having the angle be black really looked wrong to me. I feel like the angle text should also be blue, but I'll double check with design on that one.